coordinatorlib/src/main/scala/com/socrata/datacoordinator/secondary/sql/SqlSecondaryManifest.scala

```diff
@@ -208,6 +210,50 @@ class SqlSecondaryManifest(conn: Connection) extends SecondaryManifest {
   }
 
   def releaseClaimedDataset(job: SecondaryRecord): Unit = {
+    // set savepoint
+    val savepoint = conn.setSavepoint()
```

Comment:
I wonder how we might be able to refactor this so we actually get a new connection checked out each attempt... in the current setup this requires a new "universe" each time I think.  

We are already committing transactions explicitly ... perhaps we could change the universe to actually allow checking out and returning connections to make this clearer?  Unsure how big that scope would be and if it is worth it.

The goal is to avoid the case where it is failing because our connection is invalid, eg. the database restarted.  Getting a new one from the pool would ensure that.